packages/design-system/src/components/Button/Button.stories.jsx

```diff
@@ -64,13 +61,303 @@ SpinnerButton.args = {
       <Spinner /> Loading...
     </>
   ),
-  variation: 'primary',
+  variation: 'solid',
 };
 
 export const LinkButton = Template.bind({});
```

Comment:
[question] Should we remove or update this story name? I only ask because it seems a bit confusing that we have a button variation that is a "link" variation, but we also have this story called Link Button that is showcasing an anchor tag styled as a button. 

I'm not sure how useful this story so maybe we either a) remove it or b) rename it to something like "Anchor Styled as Button" 🤷🏻‍♀️

Reply:
I went ahead and updated the Story (and tests) to use "anchor" to better indicate what it's showing/testing.